main.py

```diff
@@ -136,6 +136,36 @@ def convert(load_file, dest_file):
     tf.reset_default_graph()
 
 
+def add_quant_ops(load_file, dest_file):
```

Comment:
i'm a little leery of committing code that's dead-on-arrival; could we mark this with some comments about what's broken, what's expected, etc?

Reply:
hopes... deleted